ci/Jenkinsfile

```diff
@@ -1,7 +1,13 @@
 
-parameters {
-    booleanParam(name: 'DEPLOY_PROD', defaultValue: false, description: 'deploy to production')
-}
+properties(
```

Comment:
I don't like undocumented/hidden features, let us better remove this. It also undermines the documentation purpose of the [Jenkinsfile.release](https://github.com/ElektraInitiative/PermaplanT/pull/236/files#diff-67f4db0aefcf847fe96599b66490bbc5fb3882d6f8ec188a7e0dd591b03e45d3) (there might be production builds not shown in the build log).